pennylane_pq/devices.py

```diff
@@ -141,7 +144,7 @@ def _observable_map(self):
     def _backend_kwargs(self):
         raise NotImplementedError
 
-    def __init__(self, wires=1, shots=0, *, backend, **kwargs):
+    def __init__(self, wires=1, shots=0, analytic=True, *, backend, **kwargs):
```

Comment:
It might be worth initializing with ``shots`` > 0, e.g. 1000 here

Reply:
Good catch!